modules/post/windows/gather/checkvm.rb

```diff
@@ -170,24 +183,56 @@ def xen?
       return true if service_exists?(service)
     end
 
+    return true if registry_getvaldata('HKLM\\HARDWARE\\DESCRIPTION\\System\\BIOS', 'SystemProductName') =~ /xen/i
+
     false
   end
 
   def qemu?
-    key_path = 'HKLM\HARDWARE\DEVICEMAP\Scsi\Scsi Port 0\Scsi Bus 0\Target Id 0\Logical Unit Id 0'
+    key_path = 'HKLM\\HARDWARE\\DEVICEMAP\\Scsi\\Scsi Port 0\\Scsi Bus 0\\Target Id 0\\Logical Unit Id 0'
     return true if registry_getvaldata(key_path, 'Identifier') =~ /qemu|virtio/i
 
-    key_path = 'HKLM\HARDWARE\DESCRIPTION\System\CentralProcessor\0'
+    key_path = 'HKLM\\HARDWARE\\DESCRIPTION\\System\\CentralProcessor\\0'
     return true if registry_getvaldata(key_path, 'ProcessorNameString') =~ /qemu/i
 
-    %w[HKLM\HARDWARE\ACPI\DSDT HKLM\HARDWARE\ACPI\FADT HKLM\HARDWARE\ACPI\RSDT].each do |key|
+    return true if registry_getvaldata('HKLM\\HARDWARE\\DESCRIPTION\\System', 'SystemBiosVersion') =~ /qemu/i
+    return true if registry_getvaldata('HKLM\\HARDWARE\\DESCRIPTION\\System', 'VideoBiosVersion') =~ /qemu/i
+    return true if registry_getvaldata('HKLM\\HARDWARE\\DESCRIPTION\\System\\BIOS', 'SystemManufacturer') =~ /qemu/i
+
+    %w[HKLM\\HARDWARE\\ACPI\\DSDT HKLM\\HARDWARE\\ACPI\\FADT HKLM\\HARDWARE\\ACPI\\RSDT].each do |key|
       srvvals = registry_enumkeys(key)
       return true if srvvals && srvvals.include?('BOCHS_')
     end
 
     false
   end
 
+   def parallels?
+    return true if registry_getvaldata('HKLM\\HARDWARE\\DESCRIPTION\\System', 'SystemBiosVersion') =~ /parallels/i
```

Comment:
Hey @jvoisin, thanks for enhancements 🙏 

Did some testing on Windows 11 VM running in Parallels. Found a couple issues:

1. The first time I ran the windows `checkvm` module the hypervisor was reported as `Hyper-V`
```
msf6 post(windows/gather/checkvm) > run

[*] Checking if the target is a Virtual Machine ...
[+] This is a Hyper-V Virtual Machine
[*] Post module execution completed